google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/wrappers/Validations.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigtable.data.v2.wrappers;
+
+import com.google.common.base.Preconditions;
+import javax.annotation.Nonnull;
+
+class Validations {
```

Comment:
1. Add a comment explicitly noting that this is package-private
2. Add a private constructor (as suggested by Codacy)

Reply:
done